angular/src/app/body/body.component.html

```diff
@@ -1,4 +1,4 @@
-<div *ngIf="!add && !detailedTile && newsObject">
```

Comment:
Create new component for News and use it in body to render multiple news components
use data collections for filtering to show data from all sources or by specific or to filter in that created data collection.